simtools/db_handler.py

```diff
@@ -1267,63 +1282,45 @@ def _getTaggedVersion(dbName, version="Current"):
 
         return tags["Tags"][version]["Value"]
 
-    def insertFileToDB(self, file, dbName=DB_CTA_SIMULATION_MODEL, **kwargs):
+    def insertFileToDB(self, fileName, dbName=DB_CTA_SIMULATION_MODEL, **kwargs):
         """
         Insert a file to the DB.
 
         Parameters
         ----------
+        fileName: str or Path
+            The name of the file to insert (full path).
         dbName: str
             the name of the DB
-        file: str or Path
-            The name of the file to insert (full path).
         **kwargs (optional): keyword arguments for file creation.
             The full list of arguments can be found in, \
             https://docs.mongodb.com/manual/core/gridfs/#the-files-collection
             mostly these are unnecessary though.
 
         Returns
         -------
-        file_id: gridfs "_id"
-            If the file exists, returns the "_id" of that one, otherwise creates a new one.
+        GridOut._id
+            :class:`~gridfs.grid_file.GridOut._id`
```

Comment:
As mentioned above, I don't think 
```
:class:`~gridfs.grid_file.GridOut._id`
```
is necessary, the text below it is enough.

Reply:
same comment as above, should we keep it for consistency?